### PR TITLE
Log interface for History and Persister

### PR DESCRIPTION
**Pull request by juliamourac**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature, docs update

* **What is the current behavior?** (You can also link to an open issue here)
Logging levels can only be changed through code.

* **What is the new behavior (if this is a feature change)?**
Logging level can be set through environment variables or via enpoints

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: history/conf.py [hunk 1]
```diff
@@ -8,12 +8,9 @@
...
 
-import logging
-import os
-
-LOGGER = logging.getLogger("history." + __name__)
-LOGGER.addHandler(logging.StreamHandler())
-LOGGER.setLevel(logging.INFO)
+#Logger related configuration
+levelToName = dict([(50,'CRITICAL'), (40,'ERROR'), (30,'WARNING'), (20,'INFO'), (10,'DEBUG')])
+log_level = os.environ.get("LOG_LEVEL","DEBUG")
```
Comment (mprevide) [inline history/conf.py]: i think default should be info, as you describe in readme :)

File: history/subscriber/persister.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-from dojot.module.logger import Log
 
-LOGGER = Log().color_log()
+LOGGER = Logger.Log(conf.log_level).color_log()
 
 class Persister:
 
     def __init__(self):
-        LOGGER = Log().color_log()
+        #LOGGER = Logger.Log(conf.log_level).color_log()
```
Comment (mprevide) [inline history/subscriber/persister.py]: Remove commented line

File: history/conf.py [hunk 3]
```diff
@@ -8,12 +8,9 @@
...
 # and configuration retrieval functions
 
-import logging
-import os
-
-LOGGER = logging.getLogger("history." + __name__)
-LOGGER.addHandler(logging.StreamHandler())
-LOGGER.setLevel(logging.INFO)
+#Logger related configuration
+levelToName = dict([(50,'CRITICAL'), (40,'ERROR'), (30,'WARNING'), (20,'INFO'), (10,'DEBUG')])
```
Comment (mprevide) [inline history/conf.py]: Maybe levelToName should be place into Logger.py

**Comment by mprevide** [comment]
It seems a bit strange because we have 2 log level configuration endpoints, but both set these settings in the same place. Is this correct?

File: docker/persister.docker [hunk 4]
```diff
@@ -25,4 +25,4 @@ ENV VIRTUAL_ENV="/usr/src/venv"
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 WORKDIR /usr/src/app
 
-CMD ["python", "history/subscriber/persister.py"]
+CMD ["sh", "./docker/falcon-docker-persister-entrypoint.sh", "start"]
```
Comment (mprevide) [inline docker/persister.docker]: Could you explain a little bit what this new file does and why it was created?
Maybe put some comment in the file about what it does too.

Comment (juliamourac) [reply docker/persister.docker]: This file was created with the sole purpose of running the http server in the background, so the endpoints to get and to change the log level, along with the persister service.

Comment (raulnegreiros) [reply docker/persister.docker]: @mprevide, I really don't understand wihch is the reason for this file, note that history has the same file, so @juliamourac just keep the standard. At my point of view, it's the container manager (compose, kb8s, etc) responsibility to take care of the container restart and retry mechanism, but in the sake of uniformity MAYBE makes sense this file existence. What do you think?

Comment (mprevide) [reply docker/persister.docker]: > @mprevide, I really don't understand wihch is the reason for this file, note that history has the same file, so @juliamourac just keep the standard. At my point of view, it's the container manager (compose, kb8s, etc) responsibility to take care of the container restart and retry mechanism, but in the sake of uniformity MAYBE makes sense this file existence. What do you think?

Maybe to maintain uniformity really makes sense.

**Pull request merged**